### PR TITLE
new pull request edited

### PR DESCRIPTION
**Pull request by codeserver-test**
description pull request edited

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # CSRV-11736-P2-demo-3
 CSRV-11736-P2-demo-3
```
Comment (codeserver-test) [inline README.md]: inline comment 1

**Comment by codeserver-test** [comment]
general comment 1

**Comment by codeserver-service-qa** [comment]
> > _Author: codeserver-test. Timestamp: 2019-11-05T20:16:19Z_
> 
> general comment 1

reply general comment 1

File: README.md (hunk 1 above)
Comment (codeserver-service-qa) [reply README.md]: reply comment 1

Comment (codeserver-test) [reply README.md]: another reply from customer

File: main.java [hunk 2]
```diff
@@ -3,6 +3,6 @@
 static void main(String[] args){
 
 	System.out.println("Hello World");
-	
+	//test
```
Comment (codeserver-service-qa) [inline main.java]: some comment

**Comment by codeserver-service-qa** [comment]
new comment

File: newfile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+file 3
```
Comment (codeserver-service-qa) [inline newfile.txt]: some comment

**Pull request closed without merging**